Functions.Templates/Templates/BlobTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,5 @@
+# Input bindings are passed in via param block.
+param([byte[]] $myBlob, $TriggerMetadata)
+
+# Write out the blob name and size to the information log.
+Write-Host "PowerShell Blob trigger function Processed blob! Name: $($TriggerMetadata.Name) Size: $($myBlob.Length) bytes"

```

Comment:
Do we want users to use `Write-Host` instead of `Write-Verbose`?  Also makes our templates inconsistent (see HttpTrigger template)

Reply:
`Write-Verbose` might not actually be the best one to use because you'd have to configure the host to actually display Verbose logging. Also, because in the Functions world, "Verbose" is "Trace". I opened an issue about there here:

https://github.com/Azure/azure-functions-host/issues/3701

Reply:
I will go with `Write-Host` for now.